src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActions/SuggestedAction.cs

```diff
@@ -141,39 +141,53 @@ private async Task InvokeWorkerAsync(IProgressTracker progressTracker, Cancellat
         {
             await this.ThreadingContext.JoinableTaskFactory.SwitchToMainThreadAsync(cancellationToken);
 
-            IEnumerable<CodeActionOperation> operations = null;
+            // Todo: do we want to measure the amount of time it takes get the options from the user?
```

Comment:
I think so, but reported as just that: how long did it take to get options. No harm in adding a stopwatch to measure here and reporting in the same telemetry as a property

Reply:
I will update the code here